gpu-operator/upgrade.rst

```diff
@@ -70,10 +70,10 @@ With this procedure, all existing GPU operator resources are updated inline and
    .. code-block:: console
 
       $ kubectl apply -f \
-          https://gitlab.com/nvidia/kubernetes/gpu-operator/-/raw/$RELEASE_TAG/deployments/gpu-operator/crds/nvidia.com_clusterpolicies_crd.yaml
+          https://github.com/NVIDIA/gpu-operator/blob/$RELEASE_TAG/deployments/gpu-operator/crds/nvidia.com_clusterpolicies.yaml
```

Comment:
@chenopis, I think these need to be the raw file links, right? I'm getting an error when trying to run this command

```error parsing https://github.com/NVIDIA/gpu-operator/blob/v25.3.1/deployments/gpu-operator/crds/nvidia.com_clusterpolicies.yaml: error converting YAML to JSON: yaml: line 214: mapping values are not allowed in this context```

Reply:
Yes, agreed.